pages.sv/common/cd.md

```diff
@@ -0,0 +1,20 @@
+# cd
+
+> Byt nuvarande katalog.
+> Mer information: <https://man.archlinux.org/man/cd.n>.
```

Comment:
This is an error, all pages should use this man page instead of the Arch Linux alternative.
```suggestion
> Mer information: <https://manned.org/cd>.
```

Feel free to change all the `cd` pages (all languages) in this PR :rocket:

Reply:
I've updated all `cd` pages to used manned.org in my latest commit